src/hpr_sup.erl

```diff
@@ -33,6 +31,23 @@
     modules => [I]
 }).
 
+-define(WORKER(I, Mod, Args), #{
+    id => I,
+    start => {Mod, start_link, Args},
+    restart => permanent,
+    shutdown => 5000,
+    type => worker,
+    modules => [I]
+}).
```

Comment:
The arity 3 version of ?WORKER is not used.
It looks like it should be deleted.

Reply:
👍🏻